package.json

```diff
@@ -49,6 +49,7 @@
     "dev-server": "node_modules/.bin/webpack-dev-server --content-base docs/ --inline --watch"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```

Comment:
been using it for convenience and for cleaning up code related to conditional classes, but we can do without it for now.
worth considering, though. it's relatively small (~1K unminified).

stripped it out, along with some of the CSS colors we're not using here yet - https://github.com/Clever/components/pull/36/commits/06335a20e202543f9504af8c0f4faf6f73376902